src/enums/IocServicesEnum.java

```diff
@@ -0,0 +1,15 @@
+package enums;
+
+public enum IocServicesEnum {
```

Comment:
No need to complicated too much, you can use a simple string.

Do `public static final String CLIENT_SERVICE_INSTANCE`

Reply:
solved